packages/flutter/lib/src/services/text_input.dart

```diff
@@ -2770,9 +2783,18 @@ class SystemContextMenuController with SystemContextMenuClient, Diagnosticable {
 
     ServicesBinding.systemContextMenuClient = this;
 
+    _customActionCallbacks.clear();
+    
+    for (final IOSSystemContextMenuItemData item in items) {
+      if (item is IOSSystemContextMenuItemDataCustom) {
+        _customActionCallbacks[item.callbackId] = item.onPressed;
```

Comment:
What do you think, would it be worth it to assert that there are no duplicate `item.callbackIds`? I guess it's not a problem for us if there are, and maybe users have a real reason to do something like that.

Reply:
Maybe assert if there are duplicate callbackIds AND the callbacks are not equivalent? That seems like it would very likely be a mistake on the part of the app developer.